src/helpers/normalize.js

```diff
@@ -455,7 +455,7 @@ const processNavigation = async ({ locale, nav, gatsbyHelpers }) => {
 
 exports.processNavigation = processNavigation
 
-const processGlobals = ({ globalsData, gatsbyHelpers }) => {
+const processGlobals = ({ globalsData = {}, gatsbyHelpers }) => {
```

Comment:
Does this work for Cloud Firestore as well as the Realtime Database? I know the RTDB returns `null` normally if the value isn't found in the DB, so in this case, it would not default to the empty object if passed `null`.

Reply:
Haven't tested on RTDB. Will change so it defaults it inside the method to be sure.